packages/e2e-tests/plugins/interactive-blocks/directive-each/view.js

```diff
@@ -13,6 +13,28 @@ const { state } = store( 'directive-each' );
 store( 'directive-each', {
 	state: {
 		letters: [ 'A', 'B', 'C' ],
+		eachUndefined: undefined,
+		eachNull: null,
+		eachArray: [ 'an', 'array' ],
+		eachSet: new Set( [ 'a', 'set' ] ),
+		eachString: 'str',
+		*eachGenerator() {
+			yield 'a';
+			yield 'generator';
+		},
+		eachIterator: {
+			[ Symbol.iterator ]() {
+				const vals = [ 'implements', 'iterator' ];
+				let i = 0;
+				return {
+					next() {
+						return i < vals.length
+							? { value: vals[ i++ ], done: false }
+							: { done: true };
+					},
+				};
+			},
+		},
```

Comment:
I'll admit, this is a bit esoteric, but it doesn't work in the iAPI but it seems like it could. I suspect its iterator is lost in the proxies.